database/step_test.go

```diff
@@ -160,7 +166,7 @@ func TestDatabase_Step_Validate_NoRepoID(t *testing.T) {
 		ID:      sql.NullInt64{Int64: 1, Valid: true},
 		BuildID: sql.NullInt64{Int64: 1, Valid: true},
 		Number:  sql.NullInt32{Int32: 1, Valid: true},
-		Name:    sql.NullString{String: "foo", Valid: true},
```

Comment:
I believe this was removed in error 👍 

I recommend we keep the `Name` field here and add the `Image` field as well